docs/requirement-sets/common/identity-api-requirement-sets.md

```diff
@@ -19,15 +21,17 @@ Office Add-ins run across multiple versions of Office. The following table lists
 > \* Currently, the IdentityAPI 1.3 requirement set is supported in Office on the web only for documents that are opened from Microsoft SharePoint Online and OneDrive.
 
 > [!IMPORTANT]
-> In Outlook, the Identity API requirement set isn't supported if the add-in is loaded in an Outlook.com or Gmail mailbox.
+>
+> - Regardless of the platform, the Identity API requirement set isn't supported if the Office document is opened from the **Files** tab of a Teams channel using the **Edit in Teams** option on the **Open** drop down menu.
```

Comment:
Per [style guide](https://styleguides.azurewebsites.net/Styleguide/Read?id=2700&topicid=28800), this should be dropdown

```suggestion
> - Regardless of the platform, the Identity API requirement set isn't supported if the Office document is opened from the **Files** tab of a Teams channel using the **Edit in Teams** option on the **Open** dropdown menu.
```